jina/peapods/peas/jinad.py

```diff
@@ -173,7 +173,7 @@ def filepaths(self) -> List[Path]:
             for path in self.args.upload_files:
                 try:
                     fullpath = Path(complete_path(path))
-                    paths.add(fullpath)
+                    paths.add(fullpath.stem)
```

Comment:
Any reason, this is changed?

Reply:
jinad accpets paths of List[str], here we send `List[Path]`